std/range/primitives.d

```diff
@@ -1638,6 +1638,7 @@ upTo) steps have been taken and returns $(D upTo).
 Infinite ranges are compatible, provided the parameter $(D upTo) is
 specified, in which case the implementation simply returns upTo.
  */
+deprecated("use std.algorithm.searching.count instead.")
```

Comment:
You should use the deprecated string, so it's easy to grep for:

```
// Explicitly undocumented. It will be removed in July 2018. @@@DEPRECATED_2018-07@@@
```

And of course, undocument or state in the document with `$(RED)` that it's deprecated.